.travis.yml

```diff
@@ -1,2 +1,23 @@
 language: d
+dist: trusty
 sudo: false
+
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
```

Comment:
If with these changes we would be able to use meson to build on OSX, would you mind adding OSX to the list of OSes that are tested on TravisCI? Additionally, would could expand the list of tested compilers.

It should be something like this:
```yml
os:
 - linux
 - osx
language: d
d:
 - dmd
 - dmd-beta
 - dmd-nightly
 - ldc
 - ldc-beta
```